### PR TITLE
feat(FEC-14049): add referrer to getPlaybackContext calls

### PR DESCRIPTION
**Pull request by MosheMaorKaltura**
### Description of the Changes

Please add a detailed description of the change, whether it's an enhancement or a bugfix.
If the PR is related to an open issue please link to it.

**Issue:**

**Fix:**

### CODE REVIEW

File: src/k-provider/ovp/provider.ts [hunk 1]
```diff
@@ -117,8 +119,8 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
     return typeof mediaInfo.redirectFromEntryId === 'boolean'
       ? mediaInfo.redirectFromEntryId
       : typeof this._filterOptionsConfig.redirectFromEntryId === 'boolean'
```
Comment (SivanA-Kaltura) [inline src/k-provider/ovp/provider.ts:121]: note the lint errors

File: src/k-provider/ovp/loaders/media-entry-loader.ts [hunk 2]
```diff
@@ -68,7 +68,7 @@ export default class OVPMediaEntryLoader implements ILoader {
     requests.push(OVPBaseEntryService.list(config.serviceUrl, params.ks, params.entryId, params.redirectFromEntryId, params.referenceId));
     // Use the entry id from the request result to support loading by referenceId
     const serviceEntryId = params.ks === '{1:result:ks}' ? '{2:result:objects:0:id}' : '{1:result:objects:0:id}';
-    requests.push(OVPBaseEntryService.getPlaybackContext(config.serviceUrl, params.ks, serviceEntryId));
+    requests.push(OVPBaseEntryService.getPlaybackContext(config.serviceUrl, params.ks, serviceEntryId, params.referrer));
```
Comment (SivanA-Kaltura) [inline src/k-provider/ovp/loaders/media-entry-loader.ts:71]: we should have a if (!referrer) check at some point, we probably shouldnt send undefined to the server

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: src/k-provider/ovp/provider.ts [hunk 3]
```diff
@@ -65,7 +67,7 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
           this._dataLoader.add(OVPSessionLoader, {widgetId: this.widgetId});
         }
         const redirectFromEntryId = this._getEntryRedirectFilter(mediaInfo);
-        this._dataLoader.add(OVPMediaEntryLoader, {entryId, ks, redirectFromEntryId, referenceId});
+        this._dataLoader.add(OVPMediaEntryLoader, {entryId, ks, redirectFromEntryId, referenceId, referrer: this._referrer});
```
Comment (SivanA-Kaltura) [inline src/k-provider/ovp/provider.ts:70]: is this._referrer initialized ?

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: src/k-provider/ovp/provider.ts (hunk 3 above)
Comment (MosheMaorKaltura) [reply src/k-provider/ovp/provider.ts:70]: In the constructor, It can have value or undefined

**Review by SivanA-Kaltura — APPROVED** [review]

**Pull request merged**